apps/dashboard/lib/trpc/routers/org/getInvitationList.ts

```diff
@@ -7,8 +7,14 @@ export const getInvitationList = t.procedure
   .use(requireUser)
   .use(requireOrgAdmin)
   .input(z.string())
-  .query(async ({ input: orgId }) => {
+  .query(async ({ ctx, input: orgId }) => {
     try {
+      if (orgId !== ctx.workspace?.orgId) {
```

Comment:
this should've never been passed from the frontend and still shouldn't, why don't we use ctx.workspace.orgId and remove the input altogether?